route_translator.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'actionpack', '>= 5.0.0.1', '< 5.2'
   spec.add_runtime_dependency 'activesupport', '>= 5.0.0.1', '< 5.2'
 
+  spec.add_development_dependency 'addressable', '~> 2.4'
```

Comment:
This should be a runtime dependency.

Since rails is not using this gem in runtime by default (I can see it is required by capybara, launchy, and webmock, all development dependencies), I prefer to restore URI.escape.